examples/dynamic/Makefile

```diff
@@ -0,0 +1,5 @@
+
+public/main.js: views/*
+	mkdir -p tmp
+	./node_modules/.bin/jsx -x jsx views tmp
+	./node_modules/.bin/browserify tmp/main.js -o public/main.js
```

Comment:
Good call on the `reactify` transform for Windows support. It has some strange properties, mainly that I either had to change all the template requires to include the extension, e.g. `require('./SomeTemplate.jsx')`; or I could change all extensions to `.js`. Going with the latter.

The browserify `--standalone` option was something I'd been missing. Thanks for pointing it out.

As for browserify-middleware, it looks really cool. Unfortunately, I couldn't get it to work due to some cryptic error about stream encoding. I have little patience to debug this right now.